keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -586,8 +589,11 @@ feature cannot be used. Pods using the new `VolumeSource` combined with a not
 supported container runtime version will fail to run on the node, because the
 `Mount.host_path` field is not set for those mounts.
 
-For security reasons, volume mounts should set the [`noexec`] and `ro`
-(read-only) options by default.
+For the most conservative security approach, image volumes should be mounted by
+runtimes with enabled `ro` (read-only) and `noexec` flag. Runtimes may support
```

Comment:
@saschagrunert, the `rro`, if supported (and it should be given that it's been a while), could be even safer option.

Reply:
Great hint, thank you! Updated as suggested. :pray: